### DISCUSSION TITLE
Flow triggered by an Event hook, how to block database transaction

### DISCUSSION THREAD

**Discussion by sapradhan** [directus/directus#14762] [Archive]
In the flow documentation it says 
![image](https://user-images.githubusercontent.com/5070211/181913943-a90d4a9e-66a4-4501-9a4f-45e0a920705a.png)

Could somebody point to how to prevent database operation from completing based on a condition.

**Comment by rijkvanzanten** [comment]
@nickrum?

**Comment by nickrum** [comment]
This sounds very similar to #14965 which will be possible once #15283 has been released.
